javascript/ql/test/library-tests/PropWrite/PropWritePropName.expected

```diff
@@ -1,9 +1,9 @@
+| classes.ts:4:3:4:24 | instanc ...  foo(); | instanceField |
+| classes.ts:8:15:8:35 | public  ... erField | parameterField |
 | tst.js:3:5:3:8 | x: 4 | x |
 | tst.js:4:5:6:5 | func: f ... ;\\n    } | func |
 | tst.js:7:5:9:5 | f() {\\n  ... ;\\n    } | f |
-| tst.js:12:9:12:8 | constructor() {} | constructor |
 | tst.js:13:3:15:3 | static  ... x);\\n  } | func |
-| tst.js:16:3:18:3 | f(x) {\\n ... x);\\n  } | f |
```

Comment:
Hm, this difference is due to the new charpred for `ClassMemberAsPropWrite`, I assume? I think it would still be useful to have `PropWrite`s for instance members, even if we can't say much about them. For example, a query might want to know whether there are any property writes to some property `foo`, without necessarily caring about the receiver or the RHS.

Reply:
Ok. I've pushed a change to how instance members are handled:
- Instance fields without initializers have a `PropWrite` without any RHS.
- Instance methods have a `PropWrite` without any base. Previously the base was the class object itself, which was obviously not correct.

This also means that from now on we have to be careful about using `getBase()`, since it might not always exist.

Reply:
> we have to be careful about using getBase() since it might not always exist.

That was already the case for `ClassMemberAsPropWrite`, so not a problem.